curriculum/challenges/english/14-responsive-web-design-22/learn-the-css-box-model-by-building-a-rothko-painting/step-036.md

```diff
@@ -7,7 +7,7 @@ dashedName: step-36
 
 # --description--
 
-Create a rule that targets both `.one` and `.two` and increase their `blur` effect by 1 pixel.
+Create a rule that targets both `.one` and `.two` and decrease their `blur` effect to 1 pixel.
```

Comment:
This change does not make much sense, because the `blur` effect is being _added_ - not removed. So, the amount of `blur` is _increasing_ from `0` to `1px`.

Please remove this change.

Reply:
I opened an issue on this because the blur was set in Step 35 to 2px and there was an issue with step 37. Unless the blur was reset to zero in Step 36, then since the answer is supposed to be 1px then it needs to decrease from 2 to 1. Here is a link to the lesson. Take a look at the filter/blur value for the .canvas class: https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-the-css-box-model-by-building-a-rothko-painting/step-36

Reply:
I do not see any issue. This change affects the `.one, .two` selector, and I do not see an inconsistency with the `.canvas` `filter` value.

Reply:
Ok, so I should set those words back to what they were? I'm really confused about how to do the changes. Do I make the changes and then do the normal add, commit, and push?

Reply:
Just set the words back and pushed the change